hadoop/util.sh

```diff
@@ -2,26 +2,49 @@
 
 source /usr/lib/hustler/bin/qubole-bash-lib.sh
 export PROFILE_FILE=${PROFILE_FILE:-/etc/profile}
-export HADOOP_ETC_DIR=${HADOOP_ETC_DIR:-/etc/hadoop}
+export HADOOP_ETC_DIR=${HADOOP_ETC_DIR:-/usr/lib/hadoop2/etc/hadoop}
```

Comment:
Not sure why this file is showing up - there is actually no difference in content.